.github/workflows/test.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         uses: actions/checkout@v2
 
       # Basic checkout
-      - name: Basic checkout
+      - name: Checkout basic
```

Comment:
Check status reads better like this - skim easier since lines up with `Verify ___` steps